compiler/optimizations/gpuTransforms.cpp

```diff
@@ -648,14 +658,18 @@ static VarSymbol* generateNumThreads(BlockStmt* gpuLaunchBlock,
 }
 
 static CallExpr* generateGPUCall(GpuKernel& info, VarSymbol* numThreads) {
-  CallExpr* call = new CallExpr(PRIM_GPU_KERNEL_LAUNCH_FLAT);
+  CallExpr *call = new CallExpr(PRIM_GPU_KERNEL_LAUNCH_FLAT);
```

Comment:
:) You know how to get me going with style matters.

I had been using `int *foo`, but I realized (maybe Vass pointed out?) that we use `int* foo` more widely in the compiler. So, I've been trying to switch, but old habits die hard. I think this file is mostly `int* foo`, but I see unfortunate exceptions. This'll be one more :)

Reply:
I think I'll keep it as is for now.

The argument for putting it next to the symbol is that habit it makes it less likely you'll accidentally do something like this:

`int* ptrA, ptrB`

Where you wanted `ptrB `to be a pointer too.

I'd be fine standardizing on some kind of coding style if that's what we want to do, but if so we really should be using `clang-tidy` or some linter tool or something to enforce it. Trying to enforce it through code reviews is a losing battle.